packages/manager/cypress/support/intercepts/linodes.ts

```diff
@@ -96,3 +96,27 @@ export const interceptRebootLinodeIntoRescueMode = (
     apiMatcher(`linode/instances/${linodeId}/rescue`)
   );
 };
+
+/**
+ * Intercepts GET request to retrieve a Linode's Disks and mocks response.
+ *
+ * @param linodeId - ID of Linode for intercepted request.
+ * @param disks - Array of Disks with which to mock response.
+ *
+ * @returns Cypress chainable.
+ */
+export const mockGetLinodeDisks = (
+  linodeId: number,
+  disks: Disk[]
+): Cypress.Chainable<null> => {
+  return cy.intercept(
+    'GET',
+    apiMatcher(`linode/instances/${linodeId}/disks*`),
+    {
+      results: disks.length,
+      data: disks,
+      page: 1,
+      pages: 1,
+    }
```

Comment:
```suggestion
    paginateResponse(disks)
```
Even though this works, it's probably better to stick to the `paginateResponse()` util so that we only need to touch the code in one place if the shape of our paginated responses ever changes (admittedly unlikely).